msi/tools/amazon-cloudwatch-agent.wxs

```diff
@@ -119,6 +120,9 @@
         <Component Id='TranslatorEXE' Guid='f4527006-edcb-4271-a971-039848bc8bb7' Win64='yes'>
             <File Source='config-translator.exe' KeyPath='yes' Checksum='yes'/>
         </Component>
+        <Component Id='OtelJmxJar' Guid='5af08c39-c483-4b08-8315-c99345684c4d' Win64='yes'>
+            <File Source='opentelemetry-jmx-metrics.jar' KeyPath='yes'/>
```

Comment:
does this need to be changed to `packaging/opentelemetry-jmx-metrics.jar`?

Reply:
This comes from msi zip not the repo. The repo copies to msi zip by the pre package make command